### ISSUE TITLE
Build instructions for linux

### ISSUE THREAD

**Issue by 0xfff** [hempcoin-project/THC#11]
can you provide build instructions for linux please?

**Comment by plasmacorral** [comment]
Yes, please.

**Comment by withthelemons** [comment]
This is outlined in [doc/readme-qt.rst](https://github.com/hempcoin-project/THC/blob/master/doc/readme-qt.rst) (for GUI version) and [doc/build-unix.txt](https://github.com/hempcoin-project/THC/blob/master/doc/build-unix.txt) (for daemon)

Although both are somewhat outdated, they should give you enough information

**Comment by whatSocks** [comment]
I followed the  doc/build-unix.txt instructions and got the following error 
```
alert.cpp:5:10: fatal error: 'boost/foreach.hpp' file not found
#include <boost/foreach.hpp>
         ^~~~~~~~~~~~~~~~~~~
1 error generated.
```

**Comment by withthelemons** [comment]
Make sure you have Boost libraries installed.

**Comment by whatSocks** [comment]
I installed it using homebrew

**Comment by withthelemons** [comment]
Building on macOS should be possible, but there is no official support and it might require modifying build files. Full macOS compatibility and documentation is planned.

**Comment by whatSocks** [comment]
So do you recommend instead using a debian VM?

**Comment by withthelemons** [comment]
Yes, HempCoin was tested on Debian, Ubuntu and Arch Linux.

**Comment by plasmacorral** [comment]
OK...finally got it!  Instructions where a bit dated, and ubuntu 17.10 was a no go.  I did manage to get a wallet up on ubuntu 16.04 lts.  I used the following commands, some of which are overkill and may have had nothing to do with finally getting this.

sudo apt-get update
sudo apt-get upgrade
sudo apt-get install python
sudo apt-get install git
sudo apt-get install openssl
sudo apt-get install qt4-qmake
sudo apt-get install libqt4-dev
sudo apt-get install build-essential
sudo apt-get install libboost-dev
sudo apt-get install libboost-system-dev
sudo apt-get install libboost-filesystem-dev
sudo apt-get install libboost-program-options-dev
sudo apt-get install libboost-thread-dev
sudo apt-get install libssl-dev
sudo apt-get install libdb++-dev
sudo apt-get install libdb-dev
sudo apt-get install libboost-all-dev 
sudo apt-get install libminiupnpc-dev
sudo apt-get install libqrencode-dev
git clone https://github.com/hempcoin-project/THC.git THC
cd THC
sudo nano hempcoin-qt.pro
ctl=W
search for the string "implic"
delete "-wimplicit fallback=0"
ctl-O
enter to keep filename hempcoin-qt.pro
ctl-x
qmake
make
./hempcoin-qt (failed the first time with database error)
./hempcoin-qt (profit!)

To the developers...I am new here so forgive me if this is poor form.  I ran into issues with wimplicit fallback =0 not being recognized.  I used sudo nano and edited the bitcoin-qt.pro file to remove that part.  On second launch of the gui wallet, I was in!  Hopefully this helps someone else. Also stole some of these from Nexus wallet linux instructions.
